src/main/java/com/google/sps/app/servlets/NewEventServlet.java

```diff
@@ -94,14 +94,14 @@ public Event getEvent(HttpServletRequest request){
 
         long eventID = UUIDs.generateID();
         String location = StringEscapeUtils.escapeHtml4(request.getParameter(LOCATION_PARAM));
-        Date date = parseInputDate(request);
+        Date date = parseInputDate(request.getParameter(DATE_PARAM));
         return new Event(eventID,title,location,date,estimatedCost);
     }
 
-    public Date parseInputDate(HttpServletRequest request){
+    public Date parseInputDate(String textDate){
```

Comment:
This function is in duplicated in multiple files. Maybe create a DateUtils or a ParserUtils class and move it there?
You can do that in a follow up PR.

Reply:
yes, Mauricio told me that he‘ll create a helper class, I'll add to that once he created this